examples/docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Run Remix With docker-compose
+
+very simple example to run remix app with docker and docker-compose
+
+## Preview
```

Comment:
Probably can't do a preview of this one so we can remove this section.

Reply:
If so, the `sandbox.config.json` should probably be removed too.

Reply:
i will remove `sandbox.config.json` and will add:

This example don't have Preview

to Preview section